NOTICE

```diff
@@ -17,9 +17,42 @@ under the auspices of the Eigenbase Foundation
 and released as the LucidDB project.
 
 
+================= Apache Flink =================
+Apache Flink
+Copyright 2014-2019 The Apache Software Foundation
 
 
-================= Metamarkets extendedset 1.3.10 =================
+================= Apache Hadoop =================
+Apache Hadoop
+Copyright 2009-2017 The Apache Software Foundation
+
+
+================= Apache Hive =================
+Apache Hive
+Copyright 2008-2018 The Apache Software Foundation
+
+
+================= Apache Kafka =================
```

Comment:
The source code was adapted from https://github.com/apache/kafka/blob/e554dc518eaaa0747899e708160275f95c4e525f/clients/src/main/java/org/apache/kafka/common/utils/MappedByteBuffers.java which was included in Kafka 1.0 (https://issues.apache.org/jira/browse/KAFKA-5915).